repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -71,6 +71,11 @@ object Commit {
   implicit val commitFormat: OFormat[Commit] = Json.format[Commit]
 }
 
+final case class FileCommit(commit_id: String)
```

Comment:
Let's stick to camelCase.
```suggestion
final case class FileCommit(commitId: String)
```
If `commit_id` is taken from external API, you can define custom `Json` serialization instead of automatically generated.

Reply:
Changed